Tactics/SkipProof.lean

```diff
@@ -1,3 +1,10 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Siddharth Bhat
+
+Tactic to skip proof checking for interactive use.
+-/
```

Comment:
Could you have this as a separate comment from the copyright header?

Reply:
@shigoel I'm not sure why, isn't this how we do it always? https://github.com/leanprover/lean4/blob/master/src/Init/Meta.lean#L6

Reply:
I'm not sure that file is the right example, since there's no file-level comment in it :D I guess a file such as this: https://github.com/leanprover/LNSym/blob/main/Correctness/Correctness.lean#L7-L15 shows the LNSym style. Happy to oblige.